ci-operator/config/chetna14manku/release-tests/chetna14manku-release-tests-interop-v1.10__openshift-pipelines-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  to: openshift-pipelines-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: openshift-pipelines-interop-aws
```

Comment:
Why are we using your personal fork here instead of the real repo?

Reply:
Dockerfile for containerised tests exist in my personal fork, this is the [PR](https://github.com/openshift-pipelines/release-tests/pull/137) opened for that, which is rebased with [release-v1.11](https://github.com/openshift-pipelines/release-tests/tree/release-v1.11) upstream branch